ibazel/ibazel.go

```diff
@@ -310,7 +310,7 @@ func (i *IBazel) queryForSourceFiles(query string) []string {
 	res, err := b.Query(query)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Error running Bazel %s\n", err)
-		return []string{}
+		osExit(1)
```

Comment:
WDYT about doing

```
	b := i.newBazel()
	b.WriteToStderr(true)
	b.WriteToStdout(true)

	res, err := b.Query(query)
```

here? It will return almost instantaneously and print the error to the UI which would be less terrible than parsing it out of the binarypb.

Reply:
I'm not parsing the error out of the pb. Only the query's stdout is a pb message (containing the QueryResult), whereas its stderr is just a string. I'm just collecting that string and passing it through to ibazel's stderr. In light of that, doesn't my approach make more sense than running the same query a 2nd time?